### PR TITLE
Unnest submission [OSF-6773]

### PR DESCRIPTION
**Pull request by jnayak1**
- [x] Un-nests submission from conference
- [x] Get all submissions in a conference with query params (submissions/?conference=conference_id)

-Eventually implement the include from JSONAPI standard in conference to include (submissions/?conference=conference_id) in the conference serialization. Separate PR.
-Looks like django jsonapi just [added included spec](https://github.com/django-json-api/django-rest-framework-json-api/commit/c033db9a46bdd3e77beecb66beee89419be7ab97). This is not in pypyi yet though.

### CODE REVIEW

File: meetings/conferences/urls.py [hunk 1]
```diff
@@ -12,15 +12,7 @@
     'patch': 'partial_update'
 })
 
-conference_detail = views.ConferenceViewSet.as_view({
-    'get': 'retrieve',
```
Comment (Ghalko) [inline meetings/conferences/urls.py:16]: Why remove the mapping for conference_detail?

Comment (jnayak1) [inline meetings/conferences/urls.py:16]: We have it twice @Ghalko in that file

File: meetings/conferences/serializers.py [hunk 2]
```diff
@@ -23,11 +23,8 @@ def get_links(self, obj):
                 kwargs={'pk': obj.pk},
                 request=request
             ),
-            'submissions': reverse(
-                'conferences:submissions:list',
-                kwargs={'conference_id': obj.pk},
-                request=request
-            )
+            'submissions': reverse('submissions:list',
+                                   request=request) + '?conference=' + obj.pk
```
Comment (Ghalko) [inline meetings/conferences/serializers.py]: Use `'{}?conference={}'.format(...` as a preferred method of concatenation.

File: meetings/conferences/urls.py (hunk 1 above)
Comment (Ghalko) [inline meetings/conferences/urls.py:16]: Then good catch. :)

**Pull request merged**